Google will sometimes ask for Captcha verification, causing the tests to fail


--- a/test/howdoyou-test.el
+++ b/test/howdoyou-test.el
@@ -17,7 +17,7 @@
 (ert-deftest-async test/promise-dom (done)
                    (promise-done
                     (promise-chain
-                        (howdoyou--promise-dom "https://www.google.com")
+                        (howdoyou--promise-dom "https://alpinelinux.org")
                       (then (lambda (result)
                               (should (listp result))
                               ;; (message "%s" result)
@@ -29,7 +29,7 @@
 (ert-deftest-async test/promise-curl-dom (done)
                    (promise-done
                     (promise-chain
-                        (howdoyou--curl-promise-dom "https://www.google.com")
+                        (howdoyou--curl-promise-dom "https://alpinelinux.org")
                       (then (lambda (result)
                               (should (listp result))
                               ;; (message "%s" result)
